homeassistant/components/zwave_js/discovery.py

```diff
@@ -968,7 +968,6 @@ class ZWaveDiscoverySchema:
             type={ValueType.NUMBER},
         ),
         allow_multi=True,
-        entity_registry_enabled_default=False,
```

Comment:
I think we should discuss the consequences of removing the default disabled here. It means that all these kinds of sensors, ie notification command class and number type, will be enabled by default. So there may now show up weird sensors that we don't label as diagnostic.

@AlCalzone Is there a list of all possible number type notifications to get an idea of the impact?